edx_solutions_projects/tests/test_projects.py

```diff
@@ -3,34 +3,20 @@
 """
 Run these tests: paver test_system -s lms -t edx_solutions_projects
 """
-import json
 import uuid
 
 from django.contrib.auth.models import User
 from django.core.cache import cache
-from django.test import TestCase, Client
-from django.test.utils import override_settings
-from django.utils.http import urlencode
+from django.test import TestCase
 
 from edx_solutions_projects.models import Project, Workgroup
 from edx_solutions_projects.scope_resolver import GroupProjectParticipantsScopeResolver
+from edx_solutions_api_integration.test_utils import (
+    APIClientMixin,
```

Comment:
Not any particular reason. I guess we were importing different things earlier but at the end of the day left with one class to be imported.